packages/back-end/src/models/ReportModel.ts

```diff
@@ -61,12 +61,13 @@ export async function getReportsByOrg(
   let reports = await ReportModel.find({ organization });
   // filter by project assigned to the experiment:
   if (reports && project) {
-    const allExperiments = await getExperimentsByOrganization(
-      organization,
-      project
-    );
+    const allExperiments = await getAllExperiments(organization, project);
     const expIds = new Set(allExperiments.map((e) => e.id));
-    reports = reports.filter((r) => expIds.has(r.experimentId));
+    reports = reports.filter((r) => {
+      if (r.experimentId) {
+        expIds.has(r.experimentId);
```

Comment:
Filter functions need to return true or false, but this is always returning null.  Before, it wasn't using curly braces around the function body, which means it automatically returns.

An easy fix is to remove the curly braces again

```ts
reports = reports.filter((r) => r.experimentId && expIds.has(r.experimentId));
```

Or, add return statements to your new code:

```ts
reports = reports.filter((r) => {
  if (r.experimentId) {
    return expIds.has(r.experimentId);
  }
  return false;
});
```

Reply:
DOH! Thanks